orion/pkg/repos/location_repo.go

```diff
@@ -161,26 +162,28 @@ func (lr *locationRepo) Insert(location domains.Location) error {
 	return handleSqlExecResult(result, 1, "location was not inserted")
 }
 
-func (lr *locationRepo) Publish(locIds []string) []domains.LocationErrorBody {
-	errors := make([]domains.LocationErrorBody, 0)
+func (lr *locationRepo) Publish(locIds []string) []domains.PublishErrorBody {
+	errorList := make([]domains.PublishErrorBody, 0)
 
 	for _, locId := range locIds {
 		location, err := lr.SelectByLocationId(locId)
 		if err != nil {
-			errors = append(errors, domains.LocationErrorBody{LocId: locId, Error: err})
+			errorList = append(errorList, domains.PublishErrorBody{StringId: locId, Error: err})
 			continue
 		}
 		if !location.PublishedAt.Valid {
 			now := time.Now().UTC()
 			location.PublishedAt.Scan(now)
 			err = lr.Update(locId, location)
```

Comment:
I think publishing is different from updating. This will also change `updated_at`.

Reply:
True. I think I'll replace this whole section with database transactions in a different PR anyways.